services-camel/src/main/java/edu/unc/lib/dl/services/camel/fulltext/FulltextProcessor.java

```diff
@@ -86,30 +71,16 @@ public void process(Exchange exchange) throws Exception {
             return;
         }
 
-        int retryAttempt = 0;
-
-        InputStream binaryStream = new ByteArrayInputStream(text.getBytes(StandardCharsets.UTF_8));
+        Path filepath = Paths.get(derivativeBasePath, binarySubPath + ".txt");
```

Comment:
Generally I'm not too picky about variable names, but could you rename this to `derivativePath` or something similar? It will be a little bit easier to track the difference between it and binaryPath, plus `filepath` refers to the original binary path down in `extractText`